generators/entity-server/templates/dotnetcore/src/Project.Domain/Entities/Entity.cs.ejs

```diff
@@ -73,7 +73,6 @@ namespace <%= namespace %>.Domain.Entities
             const relationshipFieldNamePascalized = relationships[idx].relationshipFieldNamePascalized;
             if (relationshipType === 'one-to-one') {
                 if(otherEntityNamePascalized === 'User') { _%>
```

Comment:
you need to remove this condition

Reply:
same for this line https://github.com/jhipster/jhipster-dotnetcore/blob/d4437035bca2aaa6e36c68bec46ced08394c01b0/generators/entity-server/templates/dotnetcore/src/Project.Domain/Entities/Entity.cs.ejs#L87

Reply:
So, @nicolas63 in case I remove the condition then the tests starts failing. I'm not able to understand how this if statement is linked with failure of tests.

Reply:
Now I have inverted the condition so that tests don't fail.

Reply:
if you reverse the conditions the bug will still be present

Reply:
can you push without the condition ? I can help you

Reply:
Sure will do it right away.

Reply:
@nicolas63 it failed [here](https://github.com/jhipster/jhipster-dotnetcore/actions/runs/3235215775/jobs/5299336240)